media-libs/libxcm/libxcm-0.5.4.recipe

```diff
@@ -65,7 +83,7 @@ defineDebugInfoPackage libxcm$secondaryArchSuffix \
 
 BUILD()
 {
-	autoreconf -fi
+	autoreconf -vfi
```

Comment:
Is this necessary?

Reply:
If I'm not mistaken in this case autoreconf is needed ...